composer.json

```
@@ -1,15 +1,34 @@
 {
-    "name": "wpcomvip/s3-media-sync",
-    "type": "wordpress-plugin",
-    "require": {
-        "php": "^8.1",
-        "composer/installers": "~1.0 || ^2.0",
-        "aws/aws-sdk-php": "~3.288.1"
-    },
-    "config": {
-        "allow-plugins": {
-            "composer/installers": true,
-            "dealerdirect/phpcodesniffer-composer-installer": true
-        }
-    }
+	"name": "wpcomvip/s3-media-sync",
+	"type": "wordpress-plugin",
+	"require": {
+		"php": "^8.1",
+		"composer/installers": "~1.0 || ^2.0",
+		"aws/aws-sdk-php": "~3.288.1"
+	},
+	"config": {
+		"allow-plugins": {
+			"composer/installers": true,
+			"dealerdirect/phpcodesniffer-composer-installer": true
+		}
+	},
+	"require-dev": {
+		"phpunit/phpunit": "^9.0",
```

```suggestion
		"phpunit/phpunit": "^9.6",
```

Since a lot of features and fixes were added in minor versions of 9.x, let's require the latest minor of v9 as the minimum.